addons/l10n_cz/models/account_move_line.py

```diff
@@ -0,0 +1,13 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import models, fields
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    def _get_rate_date(self):
+        # EXTENDS 'account'
+        self.ensure_one()
+        if self.move_id.country_code == 'CZ' and (self.move_id.taxable_supply_date or self.move_id.date) and self.currency_id:
```

Comment:
```suggestion
        if self.move_id.country_code == 'CZ':
```
I don't think it matters to check  the dates since we have a fallback on `fields.Date.context_today(self)`?
And I'm not sure why we need to check `currency_id` since it's already checked in the calling method and would not cause any Traceback?

Reply:
Make sense
Done!